bouncer/src/repo/lib/repo_license.cpp

```diff
@@ -0,0 +1,169 @@
+/**
+*  Copyright (C) 2021 3D Repo Ltd
+*
+*  This program is free software: you can redistribute it and/or modify
+*  it under the terms of the GNU Affero General Public License as
+*  published by the Free Software Foundation, either version 3 of the
+*  License, or (at your option) any later version.
+*
+*  This program is distributed in the hope that it will be useful,
+*  but WITHOUT ANY WARRANTY; without even the implied warranty of
+*  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+*  GNU Affero General Public License for more details.
+*
+*  You should have received a copy of the GNU Affero General Public License
+*  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "repo_license.h"
+
+namespace Licensing
+{
+	// definitions for class static variables
+	repo::lib::RepoUUID LicenseValidator::instanceId;
+	std::string LicenseValidator::licenseStr;
+	std::unique_ptr<Cryptolens> LicenseValidator::cryptolensHandle;
```

Comment:
Can we make sure that the library still compiles without license check? I'm pretty sure it'll fail at the moment as Cryptolens wouldn't be a part of the included library.